firefox_puppeteer/ui/menu.py

```diff
@@ -45,6 +61,15 @@ def select(self, label, item):
         """
         return self.get_menu(label).select(item)
 
+    def select_by_id(self, menu_id, item_id):
+        """Select an item in a menu.
+
+        :param menu_id: The ID of the menu, e.g., **file-menu** or **view-menu**.
+        :param item_id: The ID of the item in the menu, e.g.,
+        **menu_newNavigatorTab**.
+        """
+        return self.get_menu_by_id(menu_id).select_by_id(item_id)
```

Comment:
Keep in mind that menu items of some menus in the main menubar are getting populated when the menu is opened. If we do not open them, you might miss entries, e.g. check that by clicking the file menu, or opening it via Alt+F. Different items are shown. This also applies to dynamic lists like the bookmarks menu.